src/model/keychain.ts

```diff
@@ -70,6 +89,15 @@ export async function loadString(
   options?: Options
 ): Promise<null | string | -1 | -2> {
   try {
+    // if the data is public, try to load data from mmkv first
+    const data = loadStringMMKV(key);
```

Comment:
the comment says "if the data is public" but I don't see a check here for that (same for `loadObject`)

Reply:
there is no explicit way to check this with how this is set up currently. the idea here is that if it is private, we will get undefined and continue by checking keychain storage

we don't pass in any identified or keep track of what is private or public atm, the keychain lib handles that all under the hood